recipes/greedy/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "greedyreg" %}
+{% set version = "1.0.1" %}
+{% set ref = "f10152c5374da08ee024c4c60ef8882876bd0808" %}
```

Comment:
Where is the tag<->version relationship defined? The upstream repository doesn't seem to have tags. Please add this as a comment in the recipe.